app/src/main/java/com/example/weather/DetailsFragment.kt

```diff
@@ -86,8 +93,53 @@ class DetailsFragment : Fragment() {
         val formatter = DateTimeFormatter.ofPattern("K:mm a", Locale.ENGLISH)
         return instant.atOffset(offset).format(formatter)
     }
+
+
+    private fun displayDetailsError() {
+        binding.name.text = EMPTY_STRING
+        binding.country.text = EMPTY_STRING
```

Comment:
it's better practice to set these to View.GONE then to set the text to an empty string